src/Icon.react.coffee

```diff
@@ -0,0 +1,25 @@
+React = require 'react'
+
+Icon = React.createClass
+  displayName: 'Icon'
+
+  propTypes:
+    className: React.PropTypes.string
+    name: React.PropTypes.string.isRequired
+    style: React.PropTypes.object
+
+  getDefaultProps: ->
+    className: null
+    style: {}
+
+  render: ->
+    React.DOM.svg(
+      className: @props.className
```

Comment:
@tecnobrat as above, I can't think of a good way to establish a base `className`, like we do with `icon` on One, but I also think it may not be needed. Thoughts?